app/src/main/java/com/antosik/benchproject/app/more/viewModel/MoreViewModel.kt

```diff
@@ -0,0 +1,12 @@
+package com.antosik.benchproject.app.more.viewModel
+
+import androidx.lifecycle.ViewModel
+import com.antosik.benchproject.app.more.view.MoreFragmentNavigator
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class MoreViewModel @Inject constructor(private val moreFragmentNavigator: MoreFragmentNavigator) : ViewModel() {
+
+    fun navigateToOssLicensesMenuActivity() = moreFragmentNavigator.navigateToOssLicensesMenuActivity()
```

Comment:
There is no need to expose what this function is doing. It would be better to just name it `onLicensesClick`.

Reply:
I changed it